drivers/d3d12/rendering_device_driver_d3d12.cpp

```diff
@@ -1462,7 +1465,7 @@ RDD::TextureID RenderingDeviceDriverD3D12::_texture_create_shared_from_slice(Tex
 		uav_desc.Format = RD_TO_D3D12_FORMAT[p_view.format].general_format;
 	}
 
-	if (p_slice_type != -1) {
+	if (p_slice_type != (TextureSliceType)-1) {
```

Comment:
Isn't there a compiler flag or something else global that allows these?

Reply:
Yes, but why? This is a valid warning and might point to the real issue with integer ranges.